evm/client/chain_client.go

```diff
@@ -131,12 +130,12 @@ func NewChainClient(
 		deathDeclarationDelay,
 	)
 
-	txSender := multinode.NewTransactionSender[*types.Transaction, *SendTxResult, *big.Int, *RPCClient](
+	txSender := multinode.NewTransactionSender[*types.Transaction, struct{}, *big.Int, *RPCClient](
 		lggr,
 		chainID,
 		chainFamily,
 		multiNode,
-		NewSendTxResult,
+		func(err error) multinode.SendTxReturnCode { return 0 },
```

Comment:
This looks odd, but it matches the old `NewSendTxResult` which did not call `ClassifySendError`.
There may be more simplification available here, like dropping this func entirely, since classification already happens from inside `SendTransaction`, and the multinode itself may only be calling this classify func on it's own errors :thinking: 
But we can circle back to this later.